File: week3/.foobar.py
n = int(input())

if n % 2 == 0 and n % 3 == 0:
    print("foobar")
elif n % 2 == 0:
    print("foo")
elif n % 3 == 0:
    print("bar")

match n:
    case 0:
        print("zero")
    case 2 | 3 | 5 | 7 | 11:
        print("prime")

File: week3/.FooBar.java
import java.util.Scanner;

class FooBar {
    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        int n = sc.nextInt();

        if (n % 2 == 0 && n % 3 == 0) {
            System.out.println("foobar");
        }
        else if (n % 2 == 0) {
            System.out.println("foo");
        }
        else if (n % 3 == 0) {
            System.out.println("bar");
        }

        switch (n) {
            case 0:
                System.out.println("zero");
                break;
            case 2:
            case 3:
            case 5:
            case 7:
            case 11:
                System.out.println("prime");
        }
    }
}
